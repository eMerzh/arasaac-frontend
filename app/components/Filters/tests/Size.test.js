import { shallow } from 'enzyme'
import React from 'react'
import SelectField from 'material-ui/SelectField'
import Component from '../Size'


describe('<Size />', () => {
  it('should contain a SelectField', () => {
    const renderedComponent = shallow(<Component />)
    expect(renderedComponent.find(<SelectField />)).toBeDefined()
  })
})
