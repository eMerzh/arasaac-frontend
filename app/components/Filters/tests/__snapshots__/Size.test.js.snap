exports[`<Size /> renders correctly 1`] = `
<span
  style={
    Object {
      "display": "inline-block",
    }
  }>
  <button
    disabled={false}
    href={undefined}
    onBlur={[Function]}
    onClick={[Function]}
    onFocus={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    onMouseOut={[Function]}
    onTouchStart={[Function]}
    onTouchTap={[Function]}
    style={
      Object {
        "WebkitTapHighlightColor": "rgba(0, 0, 0, 0)",
        "background": "none",
        "border": 10,
        "boxSizing": "border-box",
        "cursor": "pointer",
        "display": "inline-block",
        "fontFamily": "Roboto, sans-serif",
        "fontSize": 0,
        "fontWeight": "inherit",
        "height": 48,
        "margin": 0,
        "muiPrepared": true,
        "outline": "none",
        "overflow": "visible",
        "padding": 12,
        "position": "relative",
        "textDecoration": "none",
        "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
        "verticalAlign": null,
        "width": 48,
        "zIndex": 1,
      }
    }
    tabIndex={0}
    type="button">
    <div
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchStart={[Function]}>
      <svg
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
        style={
          Object {
            "color": "#00bcd4",
            "display": "inline-block",
            "fill": "currentColor",
            "height": 24,
            "muiPrepared": true,
            "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
            "userSelect": "none",
            "verticalAlign": "bottom",
            "width": 24,
          }
        }
        viewBox="0 0 24 24">
        <path
          d="M14.59 8L12 10.59 9.41 8 8 9.41 10.59 12 8 14.59 9.41 16 12 13.41 14.59 16 16 14.59 13.41 12 16 9.41 14.59 8zM12 2C6.47 2 2 6.47 2 12s4.47 10 10 10 10-4.47 10-10S17.53 2 12 2zm0 18c-4.41 0-8-3.59-8-8s3.59-8 8-8 8 3.59 8 8-3.59 8-8 8z" />
      </svg>
    </div>
  </button>
  <div
    className={undefined}
    style={
      Object {
        "backgroundColor": "transparent",
        "cursor": "auto",
        "display": "inline-block",
        "fontFamily": "Roboto, sans-serif",
        "fontSize": "14px",
        "height": 72,
        "lineHeight": "24px",
        "muiPrepared": true,
        "position": "relative",
        "transition": "height 200ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
        "verticalAlign": "bottom",
        "width": 256,
      }
    }>
    <label
      className={undefined}
      htmlFor="Size"
      onTouchTap={undefined}
      style={
        Object {
          "color": "rgba(0, 0, 0, 0.3)",
          "lineHeight": "22px",
          "muiPrepared": true,
          "pointerEvents": "none",
          "position": "absolute",
          "top": 38,
          "transform": "scale(0.75) translate(0, -28px)",
          "transformOrigin": "left top",
          "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
          "userSelect": "none",
          "zIndex": 1,
        }
      }>
      Size
    </label>
    <div
      className={undefined}
      id="Size"
      onBlur={[Function]}
      onChange={[Function]}
      onFocus={[Function]}
      style={
        Object {
          "WebkitTapHighlightColor": "rgba(0,0,0,0)",
          "backgroundColor": "rgba(0,0,0,0)",
          "border": "none",
          "boxSizing": "border-box",
          "color": "rgba(0, 0, 0, 0.87)",
          "cursor": "inherit",
          "display": "block",
          "font": "inherit",
          "fontFamily": "Roboto, sans-serif",
          "fontSize": 15,
          "height": "100%",
          "marginTop": 14,
          "muiPrepared": true,
          "outline": "none",
          "padding": 0,
          "position": "relative",
          "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
          "width": "100%",
        }
      }>
      <div
        onTouchTap={[Function]}
        style={
          Object {
            "cursor": "pointer",
            "height": "100%",
            "muiPrepared": true,
            "position": "relative",
            "width": "100%",
          }
        }>
        <div
          style={
            Object {
              "boxSizing": "border-box",
              "content": "\' \'",
              "display": "table",
              "muiPrepared": true,
            }
          } />
        <div
          style={
            Object {
              "color": "rgba(0, 0, 0, 0.87)",
              "height": "56px",
              "lineHeight": "56px",
              "muiPrepared": true,
              "opacity": 1,
              "overflow": "hidden",
              "paddingLeft": 0,
              "paddingRight": 56,
              "position": "relative",
              "textOverflow": "ellipsis",
              "top": 6,
              "whiteSpace": "nowrap",
            }
          }>
          Large
        </div>
        <button
          disabled={false}
          href={undefined}
          onBlur={[Function]}
          onClick={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseEnter={[Function]}
          onMouseLeave={[Function]}
          onMouseOut={[Function]}
          onTouchStart={[Function]}
          onTouchTap={[Function]}
          style={
            Object {
              "WebkitTapHighlightColor": "rgba(0, 0, 0, 0)",
              "background": "none",
              "border": 10,
              "boxSizing": "border-box",
              "cursor": "pointer",
              "display": "inline-block",
              "fill": "#e0e0e0",
              "fontFamily": "Roboto, sans-serif",
              "fontSize": 0,
              "fontWeight": "inherit",
              "height": 48,
              "margin": 0,
              "muiPrepared": true,
              "outline": "none",
              "overflow": "visible",
              "padding": 12,
              "position": "absolute",
              "right": 0,
              "textDecoration": "none",
              "top": 8,
              "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
              "verticalAlign": null,
              "width": 48,
              "zIndex": 1,
            }
          }
          tabIndex={0}
          type="button">
          <div
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchStart={[Function]}>
            <svg
              onMouseEnter={[Function]}
              onMouseLeave={[Function]}
              style={
                Object {
                  "color": "rgba(0, 0, 0, 0.87)",
                  "display": "inline-block",
                  "fill": "inherit",
                  "height": 24,
                  "muiPrepared": true,
                  "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
                  "userSelect": "none",
                  "width": 24,
                }
              }
              viewBox="0 0 24 24">
              <path
                d="M7 10l5 5 5-5z" />
            </svg>
          </div>
        </button>
        <div
          style={
            Object {
              "borderTop": "none",
              "bottom": 1,
              "left": 0,
              "margin": "-1px 24px",
              "muiPrepared": true,
              "position": "absolute",
              "right": 0,
            }
          } />
        <div
          style={
            Object {
              "boxSizing": "border-box",
              "clear": "both",
              "content": "\' \'",
              "display": "table",
              "muiPrepared": true,
            }
          } />
      </div>
      <div
        style={
          Object {
            "display": "none",
          }
        } />
    </div>
    <div>
      <hr
        style={
          Object {
            "borderBottom": "solid 1px",
            "borderColor": "#e0e0e0",
            "borderLeft": "none",
            "borderRight": "none",
            "borderTop": "none",
            "bottom": 8,
            "boxSizing": "content-box",
            "margin": 0,
            "muiPrepared": true,
            "position": "absolute",
            "width": "100%",
          }
        } />
      <hr
        style={
          Object {
            "borderBottom": "solid 2px",
            "borderColor": "#00bcd4",
            "borderLeft": "none",
            "borderRight": "none",
            "borderTop": "none",
            "bottom": 8,
            "boxSizing": "content-box",
            "margin": 0,
            "muiPrepared": true,
            "position": "absolute",
            "transform": "scaleX(0)",
            "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
            "width": "100%",
          }
        } />
    </div>
  </div>
</span>
`;
